Conceptpower+Spring/src/main/webapp/WEB-INF/views/auth/lucene/luceneindex.jsp

```diff
@@ -125,9 +125,11 @@
 					type : "POST",
 					success: function(result) {
 					    // Poll here
-					    console.log(result);
 					    if(result.message == 'Indexer Running') {
```

Comment:
rather than checking the message you should rely on the returned http status code here.